Client/mods/deathmatch/logic/CCustomData.cpp

```diff
@@ -14,35 +14,40 @@
 
 void CCustomData::Copy(CCustomData* pCustomData)
 {
-    std::map<std::string, SCustomData>::const_iterator iter = pCustomData->IterBegin();
+    CFastHashMap<SString, SCustomData>::const_iterator iter = pCustomData->IterBegin();
     for (; iter != pCustomData->IterEnd(); iter++)
     {
-        Set(iter->first.c_str(), iter->second.Variable);
+        Set(iter->first, iter->second.Variable);
     }
 }
 
 SCustomData* CCustomData::Get(const char* szName)
 {
-    assert(szName);
```

Comment:
Why was this assertion removed?

Reply:
oh.
I removed them because originally I repalced everything with SString, because it was converted to SString anyways and there was a conversion from SString to const char* when CLuaElementDefs called CStaticFuncDefs::SetED.
I think I'll just put SString back